pkgs/development/python-modules/tesserocr/default.nix

```
@@ -0,0 +1,36 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  # build dependencies
+  cython,
+  leptonica,
+  pkg-config,
+  tesseract,
+  # extra python packages
+  pillow
+}:
+
+buildPythonPackage rec {
+  pname = "tesserocr";
+  version = "2.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0nj33mwvppacy6p5mqk9a4x26hx9ailshgad84ks60wyms6rgjiv";
+  };
+
+  buildInputs = [ leptonica tesseract ];
+  nativeBuildInputs = [ cython pkg-config ];
+  propagatedBuildInputs = [ pillow ];
+
+  # the package doesn't have any tests.
+  doCheck = false;
```

do you mind checking out from GitHub so we can pull down tests?
```suggestion
  doCheck = false;
```

tests are really important to make sure you don't get broken by bumps in upstream dependencies

---

Removed the `doCheck`, since the docs say it's `true` by default, package builds fine!

---

for buildPythonPackage, this is true